### DISCUSSION TITLE
Finish 2024-12 and Start 2025-03

### DISCUSSION THREAD

**Discussion by merks** [eclipse-simrel/simrel.build#669] [Announcements]
The 2024-12 release is final:

- https://download.eclipse.org/releases/2024-12
- https://download.eclipse.org/releases/latest

Downloads are available:

- https://www.eclipse.org/downloads/packages

---

The 2025-03 release is started:

- https://download.eclipse.org/releases/2025-03
- https://download.eclipse.org/releases/milestone

The 2025-03 staging repository exits:

- https://download.eclipse.org/staging/2025-03

The 2025-03 EPP staging products and repository are newly built against SimRel 2025-03 staging:

- https://download.eclipse.org/technology/epp/staging/
- https://download.eclipse.org/technology/epp/staging/repository/

The 2025-03 schedule is as follows:

https://github.com/eclipse-simrel/.github/blob/main/wiki/SimRel/2025-03.md

---

Orbit M0 milestones repositories for 2025-03 / 4.35 have been created.

Consult the following table for a summary overview of the updated available versions:

- https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/table.html

Aggregation - All of below

- https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/2025-03
- https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/milestone/latest

BND - Wrapped

- https://download.eclipse.org/tools/orbit/simrel/maven-bnd/2025-03
- https://download.eclipse.org/tools/orbit/simrel/maven-bnd/nightly/latest

OSGi - Direct from Maven

- https://download.eclipse.org/tools/orbit/simrel/maven-osgi/2025-03
- https://download.eclipse.org/tools/orbit/simrel/maven-osgi/milestone/latest

Native Signed

- https://download.eclipse.org/tools/orbit/simrel/maven-sign/2025-03
- https://download.eclipse.org/tools/orbit/simrel/maven-sign/milestone/latest

Legacy - Build from Source

- https://download.eclipse.org/tools/orbit/simrel/orbit-legacy/2025-03
- https://download.eclipse.org/tools/orbit/simrel/orbit-legacy/milestone/latest

---

Jetty's latest release is 12.0.15:

https://download.eclipse.org/tools/orbit/simrel/maven-jetty/release/latest

**Comment by cdietrich** [comment]
@merks seing 

error on no repo found at https://download.eclipse.org/releases/2024-12/202411221000 (in oomph)
any idea?

**Reply by merks** [reply]
Restart your IDE or use this button:

![image](https://github.com/user-attachments/assets/a396c46d-f4a3-4da9-866b-d0f36bd55954)

It sounds like you are seeing this version:

https://github.com/eclipse-oomph/oomph/commit/3cf0fee6c0e0d0ef9e4c60aba18dc0de2e631052

rather than this version:

https://github.com/eclipse-oomph/oomph/commit/3cf0fee6c0e0d0ef9e4c60aba18dc0de2e631052

**Reply by cdietrich** [reply]
hmm. thx. manually disabling stuff in available software sites and restarting, clear oomph caches has helped

**Reply by merks** [reply]
Yes, often URLs end up added to the preferences and those don't always remain valid.  😞

**Comment by pcdavid** [comment]
Not sure if this is the right place to mention this, but https://eclipseide.org/release/noteworthy/ currently still talks about 2024-09.

**Reply by merks** [reply]
I've asked @oliviergoulet5 about it.  He mentioned this issue:

https://gitlab.eclipse.org/eclipsefdn/it/websites/projects.eclipse.org/-/issues/376
